model-optimizer/extensions/middle/AddMeanScaleValues.py

```diff
@@ -64,7 +64,10 @@ def insert_pre_processing(graph: Graph, input_node: Node, node_mean_scale_values
 
         for dst in input_node.out_port(0).get_destinations():
             if dst.node.soft_get('type') != 'ShapeOf':
+                tmp = input_node.out_node(0)['fw_tensor_debug_info']
```

Comment:
Please, add a comment here why we keep the `'fw_tensor_debug_info'` in the data node of the Parameter.